### DISCUSSION TITLE
CDN connection reset

### DISCUSSION THREAD

**Discussion by guruevi** [Budibase/budibase#12160] [Help] [answered]
Hi,

I was prompted for an upgrade on the app, which I did, which seems to have broken the app. Even reverting to the previous version and re-publishing doesn't work.

I get this in the browser developer mode:
Failed to load resource: net::ERR_CONNECTION_RESET
cdn.budicloud.prod.budibase.net/<something>/budibase-client.js?v=2.11.42:1 

Failed to load resource: net::ERR_CONNECTION_RESET
 Failed to load the Budibase client
(anonymous) @ /app_…9cf7b3f94fc0177b:69

In the browser:
There was an error loading your app
The Budibase client library could not be loaded. Try republishing your app.

I've also started a new app from a template and get the same error.

Thanks!

**Comment by ConorWebb96** [comment]
Hey @guruevi,

Our engineers are currently investigating the issue, I'll post further updates whenever I know more.

Thanks!

**Reply by guruevi** [reply]
Hi,

Thanks, it seems this may be related to a web filter, it appears cdn.budicloud.prod.budibase.net has been marked as containing malware by the PAN-DB and a few others.

You can look it up here:
https://urlfiltering.paloaltonetworks.com/query/

This is fairly new as it was working ~48 hours ago.
![Unknown](https://github.com/Budibase/budibase/assets/1723625/e59123b0-b0ed-4029-a949-f1cdca5d04dc)


Thanks!

**Reply by ConorWebb96** [reply, marked as answer]
Hey @guruevi,

Sorry for the long delay on responding to this. This should be fixed now. We ended up migrating the CDN to a different URL, it shouldn't be flagged anymore.
